bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/api/BStringUtils.java

```diff
@@ -244,6 +245,83 @@ public static String getStringValue(Object value, BLink parent) {
         return refValue.stringValue(parent);
     }
 
+    /**
+     * Returns the string value of Ballerina values in expression style.
+     *
+     * @param value The value on which the function is invoked
+     * @param parent The link to the parent node
+     * @return String value of the value in expression style
+     */
+    public static String getExpressionStringValue(Object value, BLink parent) {
+        if (value == null) {
+            return "()";
+        }
+
+        BType type = TypeChecker.getType(value);
+
+        //TODO: bstring - change to type tag check
```

Comment:
Lets do this change if not done yet

Reply:
Added in https://github.com/ballerina-platform/module-ballerinai-transaction/pull/40/commits/ba3852ff7b6fab739a754330297a06682f4adab4

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/27492/commits/a69f06a43a276d1e0ae5a8e61a94f9fe317f59c5